lib/localeapp/exception_handler.rb

```diff
@@ -1,16 +1,17 @@
 module Localeapp
   class ExceptionHandler
-    def self.call(exception, locale, key, options)
+    def self.call(exception, locale, key_or_keys, options)
+      keys = Array(key_or_keys).map { |key| ERB::Util.html_escape(key.to_s) }
       Localeapp.log(exception.message)
       # Which exact exception is set up by our i18n shims
       if exception.is_a? Localeapp::I18nMissingTranslationException
-        Localeapp.log("Detected missing translation for key(s) #{key.inspect}")
+        Localeapp.log("Detected missing translation for key(s) #{keys.inspect}")
 
-        [*key].each do |key|
+        keys.each do |key|
           Localeapp.missing_translations.add(locale, key, nil, options || {})
         end
 
-        [locale, key].join(', ')
+        [locale, keys].join('.')
```

Comment:
@michaelbaudino I added a test and changed my fix.

At this point I did not take the time to understand why the expected string should be concatenated the described way. Would you mind pointing me in the right direction where I find out about the expected String format?